CedarJava/src/test/java/com/cedarpolicy/pbt/IntegrationTests.java

```diff
@@ -57,20 +58,20 @@ public void testResourceAttribute() {
         String principal = "User::\"alice\"";
         Map<String, Value> principalAttributes = new HashMap<>();
         Set<String> principalParents = new HashSet<>();
-        Entity e = new Entity(principal, principalAttributes, principalParents);
+        Entity e = new Entity(new JsonEUID("User", "alice"), principalAttributes, principalParents);
```

Comment:
Nit: It feels a little awkward to have "User" and "alice" duplicated in two places (L58 and L61). It might be cleaner if the constructor for `JsonEUID` could take as input a full EUID string and parse it, or if the `principal` string was constructed based on two other variables (one for "User" and one for "alice").

Reply:
I want to encourage use of the two argument version to avoid issues with quotes around `alice`. I don't think we can deprecate the old fn, but I'm trying to avoid using it in tests/examples.

I can save `User` and `alice` into two string and construct `principal` from them